prover/zkevm/prover/publicInput/public_input.go

```diff
@@ -109,8 +109,8 @@ func NewPublicInputZkEVM(comp *wizard.CompiledIOP, settings *Settings, ss *state
 				AbsLogNum:    getCol("loginfo.ABS_LOG_NUM"),
 				AbsLogNumMax: getCol("loginfo.ABS_LOG_NUM_MAX"),
 				Ct:           getCol("loginfo.CT"),
-				OutgoingHi:   getCol("loginfo.ADDR_HI"),
-				OutgoingLo:   getCol("loginfo.ADDR_LO"),
+				DataHi:       getCol("loginfo.DATA_HI"),
```

Comment:
@bogdanbear As a FYI. IIRC, I am the one who introduced that bug during the switch to go-corset. I changed the name of the field as I was confused by the variable name. I had thought "outgoing" meant outgoing address.